package.json

```diff
@@ -2,12 +2,16 @@
   "name": "push.js",
   "version": "0.0.1",
   "description": "A compact, cross-browser solution for Javascript desktop notifications",
-  "main": "bin/push.js",
+  "main": "dist/push.js",
   "scripts": {
-    "clean": "shx rm -rf bin",
-    "build": "shx rm -rf bin && shx mkdir -p bin && shx cp push.js bin/. && uglifyjs push.js -o bin/push.min.js",
-    "test": "karma start"
+    "clean": "shx rm -rf dist",
+    "build": "shx rm -rf dist && shx mkdir -p dist && shx cp push.js dist/. && uglifyjs push.js -o dist/push.min.js",
+    "test": "karma start",
+    "prepublish": "npm run build"
   },
+  "files": [
+    "dist"
```

Comment:
since `dist` is part of `.gitignore`, it has to be explicitly mentioned in `files` to be part of the package.